generators/app/index.js

```diff
@@ -55,15 +63,39 @@ module.exports = yeoman.Base.extend({
 
     var prompts = [
       {
+        type: 'confirm',
+        name: constants.PROP_WARNING,
+        when: function(props) {
+          if(this.bail) return false;
+          // If we find the .yo-rc.json file, show the warning
+          if(fs.existsSync(this.config.path)) {
+            this.out.warn(
+              'You are running the command within an existing project '
+              + chalk.green('"'+this._getConfigValue(constants.PROP_PROJECT_ARTIFACT_ID) + '"')
+              + ' located at ' + chalk.green('"'+path.dirname(this.config.path)+'"')+'.'
+            );
```

Comment:
I think add some output here indicating what happens if you abort or don't abort.